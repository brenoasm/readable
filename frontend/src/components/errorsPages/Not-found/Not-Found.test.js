/* eslint-disable */

import React from 'react';

import NotFound from './';

test('It matches the snapshot', () => {
  const wrapper = shallow(<NotFound />);

  expect(wrapper).toMatchSnapshot();
});
