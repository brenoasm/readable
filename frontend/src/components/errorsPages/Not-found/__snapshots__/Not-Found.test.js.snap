// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <div>
          <h2>
                    Ops...
          </h2>
          <div>
                    <p>
                              A página que você está tentando acessar não existe.
                    </p>
          </div>
</div>,
        <div>
          <Link
                    replace={false}
                    to="/"
          >
                    Ir para a página principal
          </Link>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              Ops...
</h2>,
            <div>
              <p>
                            A página que você está tentando acessar não existe.
              </p>
</div>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ops...",
            },
            "ref": null,
            "rendered": "Ops...",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p>
                A página que você está tentando acessar não existe.
</p>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "A página que você está tentando acessar não existe.",
              },
              "ref": null,
              "rendered": "A página que você está tentando acessar não existe.",
              "type": "p",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            replace={false}
            to="/"
>
            Ir para a página principal
</Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Ir para a página principal",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "Ir para a página principal",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <div>
            <h2>
                        Ops...
            </h2>
            <div>
                        <p>
                                    A página que você está tentando acessar não existe.
                        </p>
            </div>
</div>,
          <div>
            <Link
                        replace={false}
                        to="/"
            >
                        Ir para a página principal
            </Link>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                Ops...
</h2>,
              <div>
                <p>
                                A página que você está tentando acessar não existe.
                </p>
</div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ops...",
              },
              "ref": null,
              "rendered": "Ops...",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p>
                  A página que você está tentando acessar não existe.
</p>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "A página que você está tentando acessar não existe.",
                },
                "ref": null,
                "rendered": "A página que você está tentando acessar não existe.",
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              replace={false}
              to="/"
>
              Ir para a página principal
</Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Ir para a página principal",
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": "Ir para a página principal",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
